frontend/app/src/components/import/CustomImport.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div>
+    <v-row>
+      <v-col>
+        <import-source source="rotki_events">
+          <template #upload-title>
+            <strong>
+              {{ $t('import_data.custom.import_events') }}
```

Comment:
@lukicenturi for new components please `<script setup>` and also 
make sure to use `t` and `tc` from `useI18n()`,

In general, avoid using `$` prefixed properties and instead use whatever alternative exists in vue 2.7.

If we don't do this we end up giving us more work in the future.

I was looking for a general rule for `$` prefixed globals but couldn't find anything so we have to keep it in mind.